Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -54,6 +54,19 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
       .AddCodeOnlyParameter("currentScene", "")
       .MarkAsSimple();
 
+  extension
+      .AddCondition("IsSceneExist",
+                    _("Is the scene exist"),
```

Comment:
To sound more english, let's name this "DoesSceneExist" / "Does scene exist" / "Check if scene exists"  / "Scene "xx" exists"
And add a parameter "sceneName" like the action change scene below to be displayed in the condition